marshal.go

```diff
@@ -110,16 +110,22 @@ const (
 	usmStatsUnknownEngineIDs     = ".1.3.6.1.6.3.15.1.1.4.0"
 	usmStatsWrongDigests         = ".1.3.6.1.6.3.15.1.1.5.0"
 	usmStatsDecryptionErrors     = ".1.3.6.1.6.3.15.1.1.6.0"
+	snmpUnknownSecurityModels    = ".1.3.6.1.6.3.11.2.1.1.0"
+	snmpInvalidMsgs              = ".1.3.6.1.6.3.11.2.1.2.0"
+	snmpUnknownPDUHandlers       = ".1.3.6.1.6.3.11.2.1.3.0"
 )
 
 var (
-	ErrDecryption           = errors.New("decryption error")
-	ErrNotInTimeWindow      = errors.New("not in time window")
-	ErrUnknownEngineID      = errors.New("unknown engine id")
-	ErrUnknownReportPDU     = errors.New("unknown report pdu")
-	ErrUnknownSecurityLevel = errors.New("unknown security level")
-	ErrUnknownUsername      = errors.New("unknown username")
-	ErrWrongDigest          = errors.New("wrong digest")
+	ErrDecryption            = errors.New("decryption error")
+	ErrInvalidMsgs           = errors.New("invalid msgs")
```

Comment:
Let's make the message slightly more friendly with `'invalid messages`.

Reply:
```suggestion
	ErrInvalidMsgs           = errors.New("invalid messages")
```